drop-in/src/main/java/com/adyen/checkout/dropin/internal/ui/model/DropInResultContractParams.kt

```diff
@@ -6,11 +6,12 @@
  * Created by josephj on 14/2/2023.
  */
 
-package com.adyen.checkout.dropin
+package com.adyen.checkout.dropin.internal.ui.model
 
 import androidx.annotation.RestrictTo
 import com.adyen.checkout.components.core.PaymentMethodsApiResponse
-import com.adyen.checkout.dropin.service.DropInService
+import com.adyen.checkout.dropin.DropInConfiguration
+import com.adyen.checkout.dropin.DropInService
 
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
```

Comment:
This can also be an `internal class` I think

Reply:
This class is used in `DropIn.registerForDropInResult()` as a result so I guess it can be `internal` also use as a parameter for `DropIn.startPayment()`